core/base-service/transform-example.js

```diff
@@ -46,66 +43,45 @@ function validateExample(example, index, ServiceClass) {
+  const { pattern, namedParams } = result
 
-  if (staticPreview) {
-    if (!pattern && !ServiceClass.route.pattern) {
-      throw new Error(
-        `Static preview for ${
-          ServiceClass.name
-        } at index ${index} does not declare a pattern`
-      )
-    } else if (!namedParams) {
-      throw new Error(
-        `Static preview for ${
-          ServiceClass.name
-        } at index ${index} does not declare namedParams`
-      )
-    }
-    if (previewUrl) {
-      throw new Error(
-        `Static preview for ${
-          ServiceClass.name
-        } at index ${index} also declares a dynamic previewUrl, which is not allowed`
-      )
-    }
-    if (pattern === ServiceClass.route.pattern) {
-      throw new Error(
-        `Example for ${
-          ServiceClass.name
-        } at index ${index} declares a redundant pattern which should be removed`
-      )
-    }
+  if (!pattern && !ServiceClass.route.pattern) {
+    throw new Error(
+      `Example for ${
+        ServiceClass.name
+      } at index ${index} does not declare a pattern`
+    )
+  }
+  if (pattern === ServiceClass.route.pattern) {
+    throw new Error(
+      `Example for ${
+        ServiceClass.name
+      } at index ${index} declares a redundant pattern which should be removed`
+    )
+  }
 
-    // Make sure we can build the full URL using these patterns.
-    try {
-      pathToRegexp.compile(pattern || ServiceClass.route.pattern)(namedParams)
-    } catch (e) {
-      throw Error(
-        `In example for ${
-          ServiceClass.name
-        } at index ${index}, ${e.message.toLowerCase()}`
-      )
-    }
-    // Make sure there are no extra keys.
-    let keys = []
-    pathToRegexp(pattern || ServiceClass.route.pattern, keys)
-    keys = keys.map(({ name }) => name)
-    const extraKeys = Object.keys(namedParams).filter(k => !keys.includes(k))
-    if (extraKeys.length) {
-      throw Error(
-        `In example for ${
-          ServiceClass.name
-        } at index ${index}, namedParams contains unknown keys: ${extraKeys.join(
-          ', '
-        )}`
-      )
-    }
-  } else if (!previewUrl) {
+  // Make sure we can build the full URL using these patterns.
+  try {
+    pathToRegexp.compile(pattern || ServiceClass.route.pattern)(namedParams)
+  } catch (e) {
     throw Error(
-      `Example for ${
+      `In example for ${
+        ServiceClass.name
+      } at index ${index}, ${e.message.toLowerCase()}`
+    )
+  }
+  // Make sure there are no extra keys.
+  let keys = []
+  pathToRegexp(pattern || ServiceClass.route.pattern, keys)
+  keys = keys.map(({ name }) => name)
+  const extraKeys = Object.keys(namedParams).filter(k => !keys.includes(k))
+  if (extraKeys.length) {
+    throw Error(
+      `In example for ${
         ServiceClass.name
-      } at index ${index} is missing required previewUrl or staticPreview`
+      } at index ${index}, namedParams contains unknown keys: ${extraKeys.join(
+        ', '
+      )}`
```

Comment:
This diff looks strange. It's a removal of the `else` block accompanied by a change in indentation level.